nncf/experimental/post_training/compression_builder.py

```diff
@@ -42,21 +44,37 @@ def add_algorithm(self, algorithm: Algorithm) -> None:
         self.algorithms.append(algorithm)
 
     def _create_engine(self, backend: BackendType) -> Engine:
-        # TODO (Nikita Malinin): Place "ifs" into the backend-specific expandable structure
+        """
+        Creates backend-specific Engine.
+        """
         if backend == BackendType.ONNX:
             from nncf.experimental.onnx.engine import ONNXEngine  # pylint: disable=cyclic-import
```

Comment:
Add the pylint exception to [.pylintrc](https://github.com/openvinotoolkit/nncf/blob/develop/.pylintrc) and remove the inline disables.

Reply:
Added